lib/rules/no-get.js

```diff
@@ -123,6 +127,15 @@ module.exports = {
     }
 
     return {
+      ImportDeclaration(node) {
+        if (node.source.value === '@ember/object') {
+          importedGetName = importedGetName || getImportIdentifier(node, '@ember/object', 'get');
```

Comment:
Technically someone could import `get` multiple times with different names, right? Though other lint rules should prevent that, so I'm not concerned about it.

Reply:
Yeah, this rule should cover us: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-duplicates.md

Reply:
Theoretically you could turn on this rule and not that one. I don't know why someone would though :p